documentation/vespa-quick-start-multinode-aws.html

```diff
@@ -43,91 +43,47 @@ <h2 id="launch-instances">Launch instances on <a href="aws.amazon.com/ec2">AWS E
-<pre>[centos@ip-172-1-1-1 ~]$ export VESPA_HOME=/opt/vespa; export PATH=$PATH:/opt/vespa/bin</pre></li>
 
-<li><p><strong>Set the Vespa config server(s) address:</strong>
-This must be a fully qualified hostname, add the following to the <a href="setting-vespa-variables">default Vespa environment variables file</a>:  
+In this example we'll use the <samp>ip-172-1-1-1.us-east-2.compute.internal</samp> 
+instance as our administration node and all commands below are run from that node. First log on to the adminstration instance from your computer using the Public DNS record name
+(E.g <samp>ec2-18-1-1-1.us-east-2.compute.amazonaws.com</samp>). Using ssh agent forwarding will simplify the process described in the next section, given your  aws identify key file <samp>my-aws-key.pem</samp> and assuming the chosen AMI uses <samp>centos</samp> as primary login user :
 <pre>
-[centos@ip-172-1-1-1 ~]$ sudo echo "override VESPA_CONFIGSERVERS ip-172-1-1-1.us-east-2.compute.internal" >> $VESPA_HOME/conf/vespa/default-env.txt
-[centos@ip-172-1-1-1 ~]$ for i in $(cat vespa-instances.txt); do \
-        scp -i my-aws.key $VESPA_HOME/conf/vespa/default-env.txt centos@$i:tmp.env.txt; \
-  done
-
-[centos@ip-172-1-1-1 ~]$ for i in $(cat vespa-instances.txt); do \
-	ssh -i my-aws.key centos@$i mv tmp.env.txt $VESPA_HOME/conf/vespa/default-env.txt;\
-  done
-</pre>
-<p>
-Vespa requires that the <em>'hostname'</em> command returns the exact same name as registered in DNS, e.g by <em>nslookup $(hostname) |grep Name</em>.
-If it does not match you can update it on CentOs 7 by running : </p>
+$ ssh-agent add my-aws-key.pem
+$ ssh -A centos@ec2-18-1-1-1.us-east-2.compute.amazonaws.com
+</pre> 
+<p>Now we are inside the AWS EC2 cloud and we can start boostraping our system install by downloading the saws bootstrap script which will install vespa and do basic system configuration for you:</p>
 <pre>
-[centos@ip-172-1-1-1 ~]$ for i in $(cat vespa-instances.txt); do \
-	ssh -i my-aws.key centos@$i sudo hostnamectl set-hostname $i;\
-  done
+$ curl -s https://raw.githubusercontent.com/vespa-engine/sample-apps/bergum/aws-bootstrap/aws_bootstrap.sh  > aws_bootstrap.sh 
+$ for host in $(cat hosts.txt); do scp -o StrictHostKeyChecking=no aws_bootstrap.sh $host:.; done
 </pre>
-
-<li><p><strong>Start the Vespa config server:</strong>
-Start the Vespa config server on the designated admin host.
-It needs to be up and running
-before deploying your application (wait five seconds after start, then proceed to next step):</p>
+<p>Now we are ready to install and start vespa, the script expects a single hostname as argument which should be the name of the administration/configuration server, in 
+our case we chose to use <samp>ip-172-1-1-1.us-east-2.compute.internal</samp></p>
 <pre>
-[centos@ip-172-1-1-1 ~]$ sudo service vespa-configserver start
+$ for host in $(cat hosts.txt); do ssh -f $host "sudo bash aws_bootstrap.sh ip-172-1-1-1.us-east-2.compute.internal"; done  
```

Comment:
Change to `for host in $(cat hosts.txt); do ssh -f $host "sudo bash aws_bootstrap.sh $(localhost)"; done`